tools/ports/libjpeg.py

```diff
@@ -17,7 +17,7 @@ def get(ports, settings, shared):
   # Archive mirrored from http://www.ijg.org/files/jpegsrc.v9c.tar.gz.
   # We have issues where python urllib was not able to load from the www.ijg.org webserver
   # and was resulting in 403: Forbidden.
-  ports.fetch_project('libjpeg', f'https://storage.googleapis.com/webassembly/emscripten-ports/jpegsrc.v{VERSION}.tar.gz', sha512hash=HASH)
```

Comment:
I will upload the new archive to `storage.googleapis.com/webassembly/emscripten-ports` so the base URL can stay constant.  (I prefer not to rely on to many different random sites, since our CI depends on these being available with good uptime).